File: app/src/main/java/com/dhsdevelopments/potato/clientapi/message/MessageElementUnknownType.java
package com.dhsdevelopments.potato.clientapi.message;

public class MessageElementUnknownType extends MessageElement
{
    private String type;

    public MessageElementUnknownType( String type ) {
        this.type = type;
    }

    public CharSequence getSpannable() {
        return "[TYPE=" + type + "]";
    }

    @Override
    public String toString() {
        return "MessageElementUnknownType[" +
                       "type='" + type + '\'' +
                       ']';
    }
}

File: app/src/main/java/com/dhsdevelopments/potato/clientapi/message/MessageElementUnknownType.kt
package com.dhsdevelopments.potato.clientapi.message

class MessageElementUnknownType(private val type: String) : MessageElement() {

    override fun makeSpan(): CharSequence {
        return "[TYPE=$type]"
    }

    override fun toString(): String {
        return "MessageElementUnknownType[type='$type']"
    }
}
